app.js

```diff
@@ -9,17 +8,40 @@ const punkAPI = new PunkAPIWrapper();
 
 app.set('view engine', 'hbs');
 app.set('views', path.join(__dirname, 'views'));
+// Register the partials directory
 
 app.use(express.static(path.join(__dirname, 'public')));
 
-// Register the location for handlebars partials here:
-
-// ...
-
-// Add the route handlers here:
-
 app.get('/', (req, res) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res) => {
+  punkAPI
+    .getBeers()
+    .then(beersFromApi => {
+      res.render('beers', { beers: beersFromApi });
+    })
+    .catch(error => {
+      console.log(error);
+      res.status(500).send('An error occurred while fetching beers.');
+    });
+    
+});
+
+app.get('/random-beer', (req, res) => {
+  punkAPI
+    .getRandom()
+    .then(responseFromAPI => {
```

Comment:
Aamir, this method returns a single random beer. If you `console.log()` the response, you'd see that, so, there's no need to use that `randomValue` neither that `beer` variable to keep the first index of the `responseFromApi`.
You should've done it this way:
```
app.get('/random-beer', (req, res) => {
  punkAPI
    .getRandom() // .getRandom() is the method provided by punkAPI
    .then((responseFromApi) => {
      res.render('random-beer', { beer: responseFromApi });
    })
    .catch((error) => console.log(error));
});
```

Reply:
ok thank you bruno